sbtc/tests/integration/validation.rs

```diff
@@ -386,3 +400,166 @@ fn op_csv_disabled() {
         err => panic!("{err}"),
     };
 }
+
+/// This validates that a user can reclaim their funds after the locktime.
+///
+/// The test proceeds as follows:
+/// 1. Spend all the user's funds for am sBTC deposit. Check that the
```

Comment:
```suggestion
/// 1. Spend all the user's funds for a sBTC deposit. Check that the
```

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/977/commits/33fb28ccd494a137025020c0ed4a2286a0ec215e.